powerline.go

```diff
@@ -81,7 +82,15 @@ func newPowerline(cfg Config, cwd string, align alignment) *powerline {
 
 	p.theme = cfg.Themes[cfg.Theme]
 	if cfg.Shell == "autodetect" {
-		cfg.Shell = detectShell(os.Getenv("SHELL"))
+		var shellExe string
+		proc, err := process.NewProcess(int32(os.Getppid()))
+		if err == nil {
+			shellExe, _ = proc.Exe()
+		}
+		if shellExe == "" {
+			shellExe = os.Getenv("SHELL")
+		}
+		cfg.Shell = detectShell(shellExe)
```

Comment:
Looks like these changes broke some of the BSD builds. Not sure how big of a deal that is, though.

Reply:
Looks like a gopsutil bug, https://github.com/justjanne/powerline-go/pull/345 should fix it.